src/OpenTelemetry.Instrumentation.AWSLambda/TracerProviderBuilderExtensions.cs

```diff
@@ -54,10 +53,7 @@ public static class TracerProviderBuilderExtensions
         AWSMessagingUtils.SetParentFromMessageBatch = options.SetParentFromBatch;
 
         builder.AddSource(AWSLambdaWrapper.ActivitySourceName);
-        builder.SetResourceBuilder(ResourceBuilder
-            .CreateEmpty()
-            .AddTelemetrySdk()
-            .AddAttributes(AWSLambdaResourceDetector.Detect()));
+        builder.ConfigureResource(x => x.AddDetector(new AWSLambdaResourceDetector()));
```

Comment:
Is this changed triggered by OpenTelemetry.Extensions.AWS update ?

Reply:
ConfigureResource is part of the API from newer OTel SDK package.
Old version should still work, but if your user adjust resources manually it should keep changes in place.
Previous version was overriding everything.

Reply:
I think this is a great change. So far we were not able to actually use the resource functionality since we also needed some custom attributes.